webapp/sources/api-doc/components/schemas/node-inherited-properties.yml

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: CC-BY-SA-2.0
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+type: object
+required:
+  - id
+  - properties
+properties:
+  id:
+    type: string
+    description: Unique id of the node
+    format: uuid (or "root")
+    example: 9a1773c9-0889-40b6-be89-f6504443ac1b
+  properties:
+    type: array
+    description: Node properties (either set by user or filled by third party sources)
+    items:
+      type: object
+      required:
+        - name
+        - value
+      properties:
+        name:
+          type: string
+          description: Property name
+          example: datacenter
+        value:
+          format: string or JSON
+          example: AMS2
+          description: Property value (can be a string or JSON object)
+        provider:
+          type: string
+          description: Property provider (if the property is not a simple node property)
+          enum:
+          - inherited
+          - overridden
+          - plugin name like datasources
+        hierarchy:
+          type: string
+          description: An HTML description of the inheritance hierarchy for that property
+          example: "<p>from <b>Global Parameter</b>:<pre>\"start;\"</pre></p>"
```

Comment:
do we really want to promise to never change the html layout? I'm not sure it should be part of the public API.

Reply:
Well, no. But it's what was done and is already in the wild. I can emphasis that it's a non structured format. I can remove it from API too, but I can't replace it easely/quickly.

Reply:
Nonetheless, perhaps it's the correct solution

Reply:
I think I'd prefer to remove it from the API and add a structured format in the future